include/nbl/asset/IImageView.h

```diff
@@ -108,17 +108,35 @@ class IImageView : public IDescriptor
 				return false;
 			*/
 
-			// declared usages that are not a subset
-			if (!imgParams.usage.hasFlags(_params.subUsages))
+			const auto kValidUsages = core::bitflag(IGPUImage::EUF_SAMPLED_BIT)|IGPUImage::EUF_STORAGE_BIT|
```

Comment:
you don't have to do `bitflag()| | |` anymore because of `
NBL_ENUM_ADD_BITWISE_OPERATORS(IImage::E_USAGE_FLAGS)`

you can just do:
```c++
const core::bitflag<IGPUImage::E_USAGE> kValidUsages = IGPUImage::EUF_SAMPLED_BIT | IGPUImage::EUF_STORAGE_BIT | ...
```
which is more readable